fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -764,6 +764,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int insert_load_default_timeout_second = 3600; // 1 hour
 
+    /**
+     * stream load force using ip for be or not
+     */
+    @ConfField(mutable = true)
+    public static boolean stream_load_force_use_ip = false;
```

Comment:
I think it's the same issue with #20875. You can use `information_schema.load_tracking_logs` (https://github.com/StarRocks/starrocks/pull/19264) to get the error log.

Reply:
it's not the same issue with #20875  , this issue is for stream load. When fe redirect request to BE Coordinator, BE's hostname cannot be recognized by the external environment.